packages/fxa-auth-server/lib/senders/emails/layouts/fxa/index.scss

```diff
@@ -0,0 +1,52 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+@use '../../global.scss';
+
+.hidden-div {
```

Comment:
Maybe we can just call this `hidden`, I know it's specifically on a `div` here but it feels more versatile/like our other classes with that naming convention.